keps/sig-instrumentation/4827-component-statusz/README.md

```diff
@@ -225,6 +244,114 @@ metrics:/metrics
 readyz:/readyz
 sli metrics:/metrics/slis
 ```
+
+#### Data format: JSON (v1alpha1)
+
+This format is available in Alpha for programmatic access and must be explicitly requested. It is considered an alpha-level format.
+
+##### Request
+* Method: **GET**
+* Endpoint: **/statusz**
+* Header: `Accept: application/json;as=Statusz;v=v1alpha1;g=config.k8s.io`
+* Body: empty
+
+##### Response Body: `Statusz` object
+
+The response is a `Statusz` object.
+
+###### Go Struct Definition
+```go
+// Statusz is the structured response for the /statusz endpoint.
+type Statusz struct {
+	// Kind is "Statusz".
+	Kind string `json:"kind"`
+	// APIVersion is the version of the object, e.g., "config.k8s.io/v1alpha1".
+	APIVersion string `json:"apiVersion"`
+
+	// Standard object's metadata.
+	// +optional
+	Metadata metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// StartTime is the time the component process was initiated.
+	StartTime metav1.Time `json:"startTime"`
+	// UptimeSeconds is the duration in seconds for which the component has been running continuously.
+	UptimeSeconds int64 `json:"uptimeSeconds"`
```

Comment:
Seems https://github.com/kubernetes/enhancements/pull/5560#discussion_r2414540803 applies for `StartTime` as well? I'd suggest using the same `int64` here and outputting the epoch seconds, with the field renamed to `StartTimeSeconds` or something similar to reflect that.